.github/workflows/ci.yml

```diff
@@ -1,31 +1,125 @@
+# Copyright 2020 The Cockroach Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
 name: CI
 on: [push, pull_request]
 
 jobs:
   test:
+    # Execute our own tests
+    # This one will spawn 6 jobs
+    name: Main - CRDB ${{ matrix.cockroachdb-version }}, Sequelize v${{ matrix.sequelize-version }}
     strategy:
       matrix:
         sequelize-version: [5, 6]
+        cockroachdb-version: [latest, v20.2.2, v19.2.11]
       fail-fast: false
     runs-on: ubuntu-latest
-    container: cockroachdb/cockroach:latest
-    name: Sequelize v${{ matrix.sequelize-version }}
     steps:
-      -
+      - name: Start a single CockroachDB instance (${{ matrix.cockroachdb-version }}) with docker
+        env:
+          CONTAINER_ENTRYPOINT: ${{ startsWith(matrix.cockroachdb-version, 'v19') && './cockroach' || 'cockroach' }}
+        run: |
+          echo $CONTAINER_ENTRYPOINT
+          docker pull cockroachdb/cockroach:${{ matrix.cockroachdb-version }}
+          docker run -d --name roach --hostname roach -p 26257:26257 -p 8080:8080 cockroachdb/cockroach:${{ matrix.cockroachdb-version }} start-single-node --insecure
+          sudo apt update && sudo apt install wait-for-it -y
+          wait-for-it -h localhost -p 26257
+          docker exec roach bash -c "echo 'CREATE DATABASE sequelize_test;' | $CONTAINER_ENTRYPOINT sql --insecure"
+
+      - name: Checkout the repository
         uses: actions/checkout@v2
-      -
+
+      - name: Setup Node.js 12.x
         uses: actions/setup-node@v1
         with:
           node-version: 12.x
-      -
-        run: cockroach version
-      -
-        run: cockroach start-single-node --background --insecure
-      -
-        run: echo "CREATE DATABASE sequelize_test;" | cockroach sql --insecure
-      -
+
+      - name: Install dependencies
         run: npm install
-      -
-        run: npm install --save sequelize@${{ matrix.sequelize-version }}
-      -
+
+      - name: Install Sequelize v${{ matrix.sequelize-version }}
+        run: npm install --save sequelize@^${{ matrix.sequelize-version }}
+
+      - name: Run tests
         run: npm test
+
+  sequelize-postgres-integration-tests:
+    # Execute Sequelize integration tests for postgres
+    # This one will spawn 219 jobs
```

Comment:
just a question: is there a concern that this number would grow and put us over the limit?

Reply:
I doubt Sequelize will be adding a lot of new test files in the near future, so this probably won't change. The only problem would be wanting to test more versions.